build/repo.props

```diff
@@ -8,11 +8,12 @@
     <!-- These properties are use by the automation that updates dependencies.props -->
     <LineupPackageId>Internal.AspNetCore.Universe.Lineup</LineupPackageId>
     <LineupPackageRestoreSource>https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json</LineupPackageRestoreSource>
+    <_SdkVersion Condition="'$(_SdkVersion)' == ''">$([System.IO.Path]::GetFileName($([System.IO.Path]::GetDirectoryName('$(MSBuildExtensionsPath)'))))</_SdkVersion>
```

Comment:
Should we add support for this to BuildTools? It's not sustainable to copy this hack around.

/cc @natemcmaster

Reply:
https://github.com/aspnet/BuildTools/issues/679

Reply:
Do you actually need the x86 SDK? Or just the x86 runtime?

Reply:
Both

Reply:
The property way to do this then is to run the entire build for x86 and x64 separately. Mixing them in the same run is not supported. The way to run KoreBuild on windows in x86 is by setting `$env:KOREBUILD_DOTNET_ARCH=x64` cref https://github.com/aspnet/BuildTools/pull/351